### ISSUE TITLE
Setting for Overlay Layout resolution

### ISSUE THREAD

**Issue by Shaftoe62** [SHWotever/SimHub#824]
The resolution used for overlays is now dependent on the Windows Display settings.

In my case: I have a 4K monitor which is of course what I use for games.

But my Windows Display setting "Scale and Layout" is set to 200%. Otherwise the letters in normal desktop apps become way to small for me to read.

This results in a 1920 x 1080 resolution which is then also used for SimHub Overlay Layouts.

In other words: If I want to change the resolution for Overlay Layouts I have to change this Windows setting.

It would be nice if SimHub had an own setting for this.

![image](https://user-images.githubusercontent.com/53898212/118629557-fb772400-b7cd-11eb-9c8e-e288397e02be.png)

**Comment by SHWotever** [comment]
Hello,

Actually you can resize the overlay using the resize handle :
![image](https://user-images.githubusercontent.com/2207331/118674363-05168100-b7fa-11eb-8ddc-07dfeb5b9864.png)
, the rasterized parts like the map is only affected by the base resolution of the overlay (as defined in the editor) , the UI scaling should not affect it. Only the "start size" will be affected as windows changes the whole coordinate system, but it's out of control, I already tested a lot about it, there are only two working solutions with WPF/C# , totally opt out of the window scaling but it's not welcome as you pointed out (too small ui ) , or like currently where windows totally rescale the coordinate system. 

Fun gets even better when you start to scale differently each monitor :D

**Comment by Shaftoe62** [comment]
Ah, ok. I don't use multiple screens (just one very big one right in front of me) but I can imagine the headache to get that right. 

Like you said overlays are easy to resize (I make all of them in the 4k target-size since I figured that out), so it's not a big deal.

I'm just OSD prone and I want to have them scaled and positioned precisely. On the pixel! So now, when an overlay layout is roughly to my liking, i'll edit the .olayout file...
